### ISSUE TITLE
Low PCC on Xenium 5k panel

### ISSUE THREAD

**Issue by reginasyan** [KEAML-JLU/SpatialEx#8]
Dear authors,
Thank you for this exciting tool!
I have been trying to follow Tutorial 1 with my own Xenium 5K panel dataset (two slides: 130,789 and 27,474 cells). The model trains without errors, but I am getting very low PCC (~0.01–0.03) on cross-slide prediction.
**What I have tried:**

- Verified that the H&E to Xenium image registration is correct by visually overlaying cell coordinates on the H&E image

- Filtering to the 1,112 genes expressed in at least 5% of cells on both slides (reducing sparsity from 96% to 89%)

- Tuning hyperparameters: epochs (400–1000), learning rate (0.0005–0.001), prune (1000–10000), batch_size, hidden_dim, and num_neighbors

**Current setup and results:**
```
resolution = 64, image_encoder = 'uni'
num_neighbors = 7, epochs = 1000, prune = 10000, lr = 0.001
Final train loss: 2.99

Slide 1 → PCC: 0.0138, SSIM: 0.352, CMD: 0.745
Slide 2 → PCC: 0.0301, SSIM: 0.400, CMD: 0.686
```
Interestingly, a simple Ridge regression from UNI embeddings to gene expression achieves a mean PCC of 0.07 on a held-out test set, with top genes (keratins, CD74) reaching PCC 0.5–0.7. This suggests the H&E embeddings carry useful signal that the HGNN is not capturing.
**Questions:**

1. Is a patch resolution of 64 too small for UNI? Should I use 256 or larger?

2. Are there any known issues with the 5K panel or datasets of this size?

3. Are there recommended preprocessing steps or hyperparameter ranges I might be missing?

I attached my notebook below for reference. Thank you for your time!

[spatialex_xenium_5k.ipynb](https://github.com/user-attachments/files/25900539/spatialex_xenium_5k.ipynb)
